Loop/View Controllers/CarbAbsorptionViewController.swift

```diff
@@ -63,11 +63,9 @@ final class CarbAbsorptionViewController: LoopChartsTableViewController, Identif
         }
 
         navigationItem.rightBarButtonItem?.isEnabled = isOnboardingComplete
-
-        if !closedLoopStatus.isClosedLoop {
-            allowEditing = false
-        }
         
+        allowEditing = closedLoopStatus.isClosedLoop || !FeatureFlags.simpleBolusCalculatorEnabled
```

Comment:
Hm... I'm not sure this is right.  if `isClosedLoop` changes to `true` when the view loads, before the change this would not set `allowEditing` to `true`, but now it will.  Is that intentional?